ethicml/data/tabular_data/admissions.py

```diff
@@ -37,19 +37,31 @@
 }
 ```
 """
-
+from enum import Enum
+from typing import Union
 from typing_extensions import Literal
 
 from ..dataset import Dataset
 from ..util import flatten_dict
 
 __all__ = ["admissions"]
 
-AdmissionsSplits = Literal["Gender"]
 
+class AdmissionsSplits(Enum):
+    GENDER = "Gender"
+    CUSTOM = "Custom"
+
+
+VALID_STRS = Literal[tuple([e.value for e in AdmissionsSplits])]  # type: ignore[misc]
```

Comment:
You may as well use `str`. No static type checker can parse this (I think?).

Reply:
Yeah...